### PR TITLE
Added 9c-standalone tutorial on README

### PR DESCRIPTION
**Pull request by area363**

### CODE REVIEW

**Review by earlbread — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -90,3 +90,87 @@ $ docker build . -t <IMAGE_TAG>
...
+
+#### Pre-requisites
+- Docker environment: [Docker Installation Guide](https://docs.docker.com/get-started/#set-up-your-docker-environment)
+- Docker hub account: [Docker Hub Guide](https://docs.docker.com/docker-hub/)
+- AWS EC2 instance: [AWS EC2 Guide](https://docs.aws.amazon.com/ec2/index.html)
+
+#### 3. Pull your docker image to your AWS EC2 instance.
+
+```
+ex) $ sudo docker pull 9c/9c-standalone:latest
```
Comment (riemannulus) [inline README.md:132]: IMO not good at mention `sudo` in this documents. It is depending user's system environment.

**Review by riemannulus — COMMENTED** [review]
Maybe we need squash and merge when merge into main branch.

File: README.md (hunk 1 above)
Comment (area363) [reply README.md:132]: @riemannulus  Ah ok, I did put "Ubuntu Server 20.04 LTS (HVM)/t2.micro" for the example instance but I understand what you mean. I'll delete the "sudo" commands. Also, would "ex)" be unnecessary in these example commands?

Comment (riemannulus) [reply README.md:132]: LGTM about other command 👍

**Pull request closed without merging**